### PR TITLE
Forward/rel/6.4

### PR DESCRIPTION
**Pull request by dora-korpar**

### CODE REVIEW

**Comment by dora-korpar** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
@ironman-machine try

File: lib/Config.js [hunk 1]
```diff
@@ -377,9 +377,15 @@ class Config {
                 'config: invalid host for localCache. host must be a string');
             assert(typeof config.localCache.port === 'number',
                 'config: invalid port for localCache. port must be a number');
+            if (config.localCache.password !== undefined) {
```
Comment (ThibaultRiviere) [inline lib/Config.js]: Does `config.localCache.password` can be `undefined` and in this case it's not a problem to set it after ?

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     "ft_s3cmd": "cd tests/functional/s3cmd && mocha -t 40000 *.js",
     "ft_s3curl": "cd tests/functional/s3curl && mocha -t 40000 *.js",
     "ft_test": "npm-run-all -s ft_awssdk ft_s3cmd ft_s3curl ft_node",
-    "install_ft_deps": "npm install aws-sdk@2.28.0 bluebird@3.3.1 mocha@2.3.4 mocha-junit-reporter@1.11.1 tv4@1.2.7",
+    "install_ft_deps": "npm install aws-sdk@2.28.0 bluebird@3.3.1 mocha@2.3.4 mocha-junit-reporter@1.11.1 tv4@1.2.7 zombie@5.0.5",
```
Comment (LaurenSpiegel) [inline package.json]: I don't think we need zombie in rel/7.0 or master. It was a dependency for the bucket website tests that we ended up removing through a refactor by @electrachong I believe.

File: lib/Config.js (hunk 1 above)
Comment (rahulreddy) [inline lib/Config.js]: @ThibaultRiviere Redis password is optional, so the config check will apply only when it's defined.

**Comment by rahulreddy** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
@ironman-machine try

**Pull request merged**